lib/Events/NewTOSCreatedEvent.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * @copyright Copyright (c) 2017 Bjoern Schiessle <bjoern@schiessle.org>
+ *
+ * @author Arthur Schiwon <blizzz@arthur-schiwon.de>
+ * @author Bjoern Schiessle <bjoern@schiessle.org>
+ * @author Roeland Jago Douma <roeland@famdouma.nl>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+namespace OCA\TermsOfService\Events;
+
+use OCP\EventDispatcher\Event;
+
+/**
+ * Class MoveToTrashEvent
+ *
+ * Event to allow other apps to disable the trash bin for specific files
+ *
+ * @package OCA\Terms_Of_Service\Events
+ */
+class NewTOSCreatedEvent extends Event {
```

Comment:
Following the existing naming convention this should be:
```suggestion
class TermsCreatedEvent extends Event {
```